api/notd/manager.py

```diff
@@ -113,11 +113,13 @@ async def retrieve_ui_data(self, startDate: datetime.datetime, endDate: datetime
             orders=[RandomOrder()],
             limit=1
         )
+        transferCount = await self.retriever.get_token_transfer_count(startDate=startDate, endDate=endDate)
         return UiData(
             highestPricedTokenTransfer=highestPricedTokenTransfers[0],
             mostTradedTokenTransfers=mostTradedTokenTransfers,
             randomTokenTransfer=randomTokenTransfers[0],
             sponsoredToken=self.get_sponsored_token(),
+            transferCount = transferCount
```

Comment:
i owuld expect this to be added to the models too otherwise it wont be returned from the api. Are you able to run the api locally and hit this endpoint and see this field?